action-server/src/listeners/dbListeners.ts

```diff
@@ -78,7 +78,7 @@ async function runAction(payload: ActionRunInsertedPayload) {
   }
 
   const duration = Math.round(performance.now() - start);
-  const status = taskError || run?.errors ? "failed" : "complete";
+  const status = taskError || run?.errors ? "failed" : "success";
```

Comment:
I don't see a similar update for setting the status to `incomplete`. I would expect to see such a line around `worker::runAction`. Is Github hiding it from me or does the Action Server not post when it starts _unning an action.

Reply:
You are correct, This came up yesterday during Cody's work. Dan noted in the Action subteam thread that we plan to resolve it in V2 if that is okay with you?

Reply:
If you're not logging it at all? Then yes, it can wait until v2.